### PR TITLE
Tickets shopping cart

### PR DESCRIPTION
**Pull request by Ketch92**

### CODE REVIEW

File: src/main/java/core/model/ShoppingCart.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_cart")
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @OneToMany
+    @JoinTable(name = "tickets_cats",
```
Comment (vlykhvar) [inline src/main/java/core/model/ShoppingCart.java]: cats?)

**Review by vlykhvar — APPROVED** [review]

File: src/main/java/core/model/ShoppingCart.java (hunk 1 above)
Comment (Ketch92) [reply src/main/java/core/model/ShoppingCart.java]: Why not?) Thanks for review!)

File: src/main/java/core/service/impl/ShoppingCartServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    private ShoppingCartDao shoppingCartDao;
+    @Inject
+    private TicketDao ticketDao;
+    
+    @Override
+    public void addSession(MovieSession movieSession, User user) {
+        Ticket ticket = new Ticket();
+        ticket.setMovieSession(movieSession);
+        ticket.setUser(user);
+        ticket = ticketDao.add(ticket);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/service/impl/ShoppingCartServiceImpl.java]: let's move this after getting shopping cart

**Review by dmytro-hryhoruk — APPROVED** [review]

File: src/main/java/core/service/impl/ShoppingCartServiceImpl.java (hunk 2 above)
Comment (Ketch92) [reply src/main/java/core/service/impl/ShoppingCartServiceImpl.java]: image

Comment (Ketch92) [reply src/main/java/core/service/impl/ShoppingCartServiceImpl.java]: I put the getting the ShoppingCart to before adding the ticket to DB

**Pull request merged**